src/test/scala/com/ambiata/poacher/hdfs/HdfsSpec.scala

```diff
@@ -51,4 +58,22 @@ class HdfsSpec extends Specification { def is = s2"""
     v ==== 1
   }
 
+  def moveFile = prop((source: HdfsTemporary, dest: HdfsTemporary, contents: String) => (for {
```

Comment:
Should we add one where the destination directory already exists as well? /notsure